tests/test_tmux_osc_bridge_regression.py

```
@@ -0,0 +1,78 @@
+Regression checks for tmux OSC bridge wiring.
+"""
+
+from __future__ import annotations
+
+import subprocess
+from pathlib import Path
+
+
+def repo_root() -> Path:
+    result = subprocess.run(
+        ["git", "rev-parse", "--show-toplevel"],
```

Ruff (0.15.2)

[error] Starting a process with a partial executable path

(S607)